.github/workflows/push_pr.yaml

```diff
@@ -0,0 +1,56 @@
+name: Push/PR
+
+on:
+  push:
+    branches:
+      - main
+      - master
+      - renovate/**
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+
+  static-analysis:
```

Comment:
The workflow's name doesn't say `_reusable` but, do you think we could leverage the coreint-automation's workflow for static analysis? https://github.com/newrelic/coreint-automation/blob/main/.github/workflows/_static_analysis.yaml

Reply:
Didn't use this workflow because we would need to modify it to accept arguments for `go-version-file` and `target-directory`.
`nri-statsd` does not have `go.mod` in the default place where the rest of repos have it.

Reply:
I see this is also documented in the status&schedule document, thanks! 👍